egs/wsj/s5/utils/slurm.pl

```diff
@@ -95,6 +95,12 @@ ()
   exit 1;
 }
 
+sub exec_command {
+  my $command = join ' ', @_;
+  # To get the actual exit value, shift right by eight bits.
+  ($_ = qx{$command 2>&1}, $? >> 8);
```

Comment:
This PR is mostly OK, but can you please rewrite the exec_command function to be easier to understand and to use backticks or "system" instead of the very obscure 'qx' operator?